_posts/en/newsletters/2019-11-06-newsletter.md

```diff
@@ -0,0 +1,152 @@
+    advantages.  Naumenko's email requests feedback on which method would
+    be preferred by implementers of both full nodes and lightweight
+    clients.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #16943][] adds a `generatetodescriptor` RPC that allows
+  new blocks generated during testing (e.g. in regtest mode) to pay a
+  script represented by an [output script descriptor][].  Prior to this
+  change, only the `generatetoaddress` RPC was available and it can only
+  pay P2PKH, P2SH, or P2WSH addresses.
+
+- [C-Lightning #3220][] begins always creating signatures with a low *r*
+  value, reducing the maximum size of the signature by one byte and
+  saving about 0.125 vbytes on average per C-Lightning peer in onchain
+  transactions.  Bitcoin Core also previously adopted this change to its
+  wallet's signature-generation code (see [Newsletter #8][news8 lowr]).
+
+- [LND #3558][] synthesizes a unified policy for any case where two
+  particular nodes have multiple channels open between them and then
+  uses this synthetic policy when considering routing through any of
+  those channels.  [BOLT4][bolt4 non-strict rec] recommends that multiple channels
+  between the same nodes should all use the same policy, but this
+  doesn't always happen, so this change tries to determine "the greatest
+  common denominator of all policies" between the nodes.  Using a single
+  policy reduces the number of routes the node needs to evaluate when
+  making a payment.
+
+- [LND #3556][] adds a new `queryprob` RPC that returns the expected
+  probability that a payment would succeed given a particular
+  destination node and amount.  This replaces functionality previously
+  removed from the `querymc` RPC.
+
+- [BOLTs #660][] updates [BOLT1][] to reserve Type-Length-Value (TLV)
+  type identifiers less than 2<sup>16</sup> for types defined in in the
+  LN specification (the BOLT documents).  The remaining values may be
+  used freely as custom records by any LN implementation.  The updated
+  specification also now provides guidance on how to select numbers for
+  custom record types.
+
+{% include linkers/issues.md issues="16943,3558,3556,660,3220,688" %}
+[bitcoin core 0.19.0]: https://bitcoincore.org/bin/bitcoin-core-0.19.0/
+[bolt4 non-strict rec]: https://github.com/lightningnetwork/lightning-rfc/blob/master/04-onion-routing.md#recommendation
+[news37 addrv2]: /en/newsletters/2019/03/12/#version-2-addr-message-proposed
+[news8 lowr]: /en/newsletters/2018/08/14/#bitcoin-core-wallet-to-begin-only-creating-low-r-signatures
+[naumenko addr relay]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-October/017428.html
```

Comment:
nit on ordering

Reply:
I don't usually put the links in any order---I just add new links to the list as I look them up, or duplicate a previous entry and tweak it as necessary.

Reply:
Sounds good